lib/sms.rb

```diff
@@ -0,0 +1,35 @@
+require 'twilio-ruby'
+
+class Sms
+
+
+
+  def initialize(client_klass = Twilio::REST::Client)
+    @client_klass = client_klass
+    @client = get_client
+
+    @from = +441740582013
```

Comment:
Style/NumericLiterals: Separate every 3 digits in the integer portion of a number with underscores(_). (https://github.com/bbatsov/ruby-style-guide#underscores-in-numerics)